src/homelogstore/log_store.cpp

```diff
@@ -388,6 +391,14 @@ sisl::status_response HomeLogStore::get_status(const sisl::status_request& reque
     response.json["truncation_parallel_to_writes?"] = m_safe_truncation_boundary.active_writes_not_part_of_truncation;
     response.json["logstore_records"] = m_records.get_status(request.verbose_level);
     response.json["logstore_sb_first_lsn"] = m_logdev.m_logdev_meta.store_superblk(m_store_id).m_first_seq_num;
+
+    log_dump_req dump_req{};
+    if (!request.next_cursor.empty()) { dump_req.start_seq_num = std::stoul(request.next_cursor); }
+    dump_req.end_seq_num = dump_req.start_seq_num + request.batch_size;
+    homestore::log_dump_verbosity verbose_level = homestore::log_dump_verbosity::HEADER;
+    if (request.json.contains("log_content")) { verbose_level = homestore::log_dump_verbosity::CONTENT; }
+    dump_req.verbosity_level = verbose_level;
+    response.json.update(dump_log_store(dump_req));
```

Comment:
dump_log_store(const log_dump_req& dump_req) function returns content. Do we need to call the function here in get_status()

Reply:
There exist APIs for dump logstore family (dump_logstore_family()) and logstore (dump_logstore_by_id). If we decide to call dump_log_store here, then we will probably need to retire /api/v1/logstore/:family/dump and api/v1/logstore/:family/:logstore/dump because they are gonna be verbose.